response_operations_ui/views/accounts.py

```diff
@@ -0,0 +1,204 @@
+from itsdangerous import URLSafeSerializer, BadSignature, BadData, SignatureExpired
+
+from response_operations_ui.forms import RequestAccountForm, CreateAccountForm
+from response_operations_ui.controllers import uaa_controller
+from response_operations_ui.controllers.notify_controller import NotifyController
+from response_operations_ui.common import token_decoder
+from response_operations_ui.exceptions.exceptions import NotifyError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+account_bp = Blueprint('account_bp', __name__, static_folder='static', template_folder='templates')
+
+
+@account_bp.route('/request-new-account', methods=['GET'])
+def get_request_new_account(form_errors=None, email=None):
+    form = RequestAccountForm(request.form)
+    template_data = {
+        "error": {
+            "type": form_errors
+        }
+    }
+    return render_template('request-new-account.html', form=form, data=template_data, email=email)
+
+
+@account_bp.route('/request-new-account', methods=['POST'])
+def post_request_new_account():
+    form = RequestAccountForm(request.form)
+    form.email_address.data = form.email_address.data.strip()
+    email = form.data.get('email_address')
+
+    if form.validate():
+        local_part, domain_part = email.rsplit('@', 1)
+        logger.info('This is the domain part', domain_part=domain_part)
+        if domain_part != 'ons.gov.uk' and domain_part != 'ext.ons.gov.uk' and domain_part != 'ons.fake':
+            logger.info('Account requested for non-ONS email address')
+            errors = {'email_address': ['Not a valid ONS email address']}
+            return get_request_new_account(form_errors=errors, email=email)
+    
+        password = form.data.get('password')
+        admin_password = app.config['CREATE_ACCOUNT_ADMIN_PASSWORD']
+        if password != admin_password:
+            logger.info('Invalid password provided for create account')
+            errors = {'password': ['Incorrect admin secret']}
+            return get_request_new_account(form_errors=errors, email=email)
+
+        return send_create_account_email(email)
+
+    errors = {'email_address': ['Not a valid ONS email address']}
+    return get_request_new_account(form_errors=errors, email=email)
+
+
+@account_bp.route('/account-exists', methods=['GET'])
+def request_new_account_account_exists():
+    encoded_email = request.args.get('email')
+
+    if encoded_email is None:
+        logger.error('No email parameter supplied')
+        return redirect(url_for('account_bp.get_request_new_account'))
+
+    try:
+        email = URLSafeSerializer(app.config['SECRET_KEY']).loads(encoded_email)
+    except BadSignature:
+        logger.error('Unable to decode email from URL', encoded_email=encoded_email)
+        abort(404)
+
+    return render_template('request-new-account-exists.html', email=email)
+
+
+@account_bp.route('/request-new-account-check-email', methods=['GET'])
```

Comment:
Same comment as the new_account_exists endpoint.  This doens't need to exist.  Instead of redirecting to a new endpoint, you can just render this template